src/actions/TransactionDetailsActions.js

```diff
@@ -49,3 +57,103 @@ export const getSelectedWallet = (state: RootState): EdgeCurrencyWallet => {
   const { currencyWallets = {} } = state.core.account
   return currencyWallets[selectedWalletId]
 }
+
+export const updateRbfTransaction = (transaction: ?EdgeTransaction, error: ?Error): Actions => ({
```

Comment:
Doesn't seem like there's any reason for these redux actions. Everything related to RBF can simply be a call directly to the core to send the transaction

Reply:
After discussing with @paullinator, I'll refactor out the redux portion of this commit and force push the initial commit for this work with the refactored commit.